lib/models/movie.dart

```diff
@@ -1,36 +1,78 @@
 class Movie {
   final String title;
   final String originalTitle;
+  final String originalLanguage;
+  final int id;
+  final bool adult;
   final String releaseDate;
   final String overview;
   final double voteAverage;
-  final List<String> genres;
+  final List<int> genres;
   final String posterPath;
   final String backdropPath;
+  final double popularity;
+  final bool video;
+  final int voteCount;
 
-  const Movie(
-      {required this.title,
-      required this.originalTitle,
-      required this.releaseDate,
-      required this.overview,
-      required this.voteAverage,
-      required this.genres,
-      required this.posterPath,
-      required this.backdropPath});
+  String get posterUrl => 'https://image.tmdb.org/t/p/w500$posterPath';
+
+  String get backdropUrl => 'https://image.tmdb.org/t/p/w500$backdropPath';
+
+  const Movie({
+    required this.title,
+    required this.originalTitle,
+    required this.originalLanguage,
+    required this.id,
+    required this.adult,
+    required this.releaseDate,
+    required this.overview,
+    required this.voteAverage,
+    required this.genres,
+    required this.posterPath,
+    required this.backdropPath,
+    required this.popularity,
+    required this.video,
+    required this.voteCount,
+  });
 
   factory Movie.mockMovie() {
     return const Movie(
       title: 'The Super Mario Bros. Movie',
       originalTitle: 'The Super Mario Bros. Movie',
+      originalLanguage: 'en',
+      id: 1,
+      adult: false,
       releaseDate: '2023-04-05',
       overview: 'While working underground to fix a water main, '
           'Brooklyn plumbers—and brothers—Mario and Luigi are transported down '
           'a mysterious pipe and wander into a magical new world. But when the '
           'brothers are separated, Mario embarks on an epic quest to find Luigi.',
       voteAverage: 7.8,
-      genres: ['Animation', 'Family', 'Adventure', 'Fantasy', 'Comedy'],
+      genres: [],
```

Comment:
Why the mock movie have an empty list, here you need to have the IDs from the names that you are removing, right?

Reply:
True I didn't see that because the mock movie isn't used. I will change that